t/17-basetest.t

```diff
@@ -116,6 +116,28 @@ subtest parse_serial_output => sub {
     is($basetest->{result}, 'ok', 'test result set to ok');
     is($message, 'CPU soft lockup detected - Serial error: Serial to match', 'log message matches output');
 
+    $basetest->{result} = 'softfail';
+    $basetest->{serial_failures} = [
+        {type => 'info', message => 'CPU soft lockup detected', pattern => qr/Serial/},
+    ];
+    $basetest->parse_serial_output_qemu();
+    is($basetest->{result}, 'softfail', 'test result stays at softfail on ok match');
+    is($message, 'CPU soft lockup detected - Serial error: Serial to match', 'log message matches output');
+
+    $basetest->{result} = 'fail';
+    $basetest->{serial_failures} = [
+        {type => 'info', message => 'CPU soft lockup detected', pattern => qr/Serial/},
+    ];
+    $basetest->parse_serial_output_qemu();
+    is($basetest->{result}, 'fail', 'test result stays at fail on ok match');
+    is($message, 'CPU soft lockup detected - Serial error: Serial to match', 'log message matches output');
```

Comment:
there is still three times repeated "log message matches output"

Reply:
I removed that lines as that is already tested before.